server/app/views/components/ButtonStyles.java

```diff
@@ -0,0 +1,65 @@
+package views.components;
+
+import views.style.BaseStyles;
+import views.style.StyleUtils;
+
+/**
+ * A collection of styles for buttons throughout CiviForm. Importantly, the public style Strings in
+ * this file should be **generalizable**, meaning they are not designed for a particuar use case.
+ * Styles should not be called i.e. BUTTON_SUBMIT_APPLICATION for example. If a specific change is
+ * required for a use case, use a general style and add more classes in the client code.
+ */
+public final class ButtonStyles {
+  /**
+   * Base styles for buttons in the applicant UI. This is missing a specified text size, so that
+   * should be added by other button style constants that use this as a base.
+   */
+  private static final String BUTTON_BASE =
+      StyleUtils.joinStyles(
+          "block", "py-2", "text-center", "rounded-full", "border", "border-transparent");
+
+  /** Base styles for buttons with a solid background color. */
```

Comment:
**optional:** maybe specify it's solid blue specifically

Reply:
Done.